docs/docs/Develop/install-custom-dependencies.mdx

```diff
@@ -61,45 +66,53 @@ source YOUR_LANGFLOW_VENV/bin/activate
 uv pip install langflow matplotlib
 ```
 
-If you're working within a cloned Langflow repository, add dependencies with `uv add` to reference the existing `pyproject.toml` file:
+If you're working within a cloned Langflow repository, add dependencies with `uv add` to reference the existing `pyproject.toml` files:
 
 ```bash
-uv add langflow matplotlib
+uv add matplotlib
 ```
 
+The `uv add` commands automatically update the `uv.lock` file in the appropriate location.
+
 ## Add dependencies to the Langflow codebase
 
 When contributing to the Langflow codebase, you might need to add dependencies to Langflow.
 
-Langflow uses a workspace with two packages:
-
-* The `main` package (root level): For end-user features and main application code
-* The `base` package (in `src/backend/base`): For core functionality and shared code
-
-Dependencies can be added in different groups:
+Langflow uses a workspace with two packages, each with different types of dependencies.
 
-* Regular dependencies: Core functionality needed to run the package
-* Development dependencies: Tools for testing, linting, or debugging are added in the `[dependency-groups.dev]` section
-* Optional dependencies: Features that users can optionally install are added in the`[project.optional-dependencies]`
+To add a dependency to the `main` package, navigate to the project root and run:
+```bash
+uv add matplotlib
+```
```

Comment:
~~~suggestion
To add a dependency to the `main` package, run `uv add DEPENDENCY` from the project root.
For example:

```bash
uv add matplotlib
```
~~~